xdsl/dialects/llvm.py

```diff
@@ -673,31 +677,74 @@ def get(ptr_type: LLVMPointerType | None = None):
 
 
 @irdl_op_definition
-class LLVMExtractValue(IRDLOperation):
+class ExtractValueOp(IRDLOperation):
+    """
+    https://mlir.llvm.org/docs/Dialects/LLVM/#llvmextractvalue-mlirllvmextractvalueop
+    """
+
     name = "llvm.extractvalue"
 
     position: DenseArrayBase = attr_def(DenseArrayBase)
-    container: Operand = operand_def(AnyAttr())
+    container: Operand = operand_def(Attribute)
```

Comment:
```suggestion
    container: Operand = operand_def()
```

I am getting confused again with our syntax, shouldn't this work as well?

Reply:
All three are equivalent, you're right, as well as `container = operand_def()`

Reply:
I don't really have a strong preference for either quite yet, maybe one will form over time